File: src/main/java/no/nav/sbl/dialogarena/modiasyforest/controller/domain/adresse/Gateadresse.java
package no.nav.sbl.dialogarena.modiasyforest.controller.domain.adresse;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

import java.math.BigInteger;

@Data
@Accessors(fluent = true)
@EqualsAndHashCode
public class Gateadresse extends StrukturertAdresse {
    public String poststed;
    public String postnummer;
    public BigInteger husnummer;
    public String husbokstav;
    public String kommunenummer;
    public String gatenavn;
    public String bolignummer;
    public BigInteger gatenummer;
}

File: src/main/kotlin/no/nav/syfo/controller/user/domain/adresse/Gateadresse.kt
package no.nav.syfo.controller.user.domain.adresse

import java.math.BigInteger

data class Gateadresse(
    val poststed: String? = null,
    val postnummer: String? = null,
    val husnummer: BigInteger? = null,
    val husbokstav: String? = null,
    val kommunenummer: String? = null,
    val gatenavn: String? = null,
    val bolignummer: String? = null,
    val gatenummer: BigInteger? = null
) : StrukturertAdresse()
